src/main/java/hudson/plugins/active_directory/ActiveDirectorySecurityRealm.java

```diff
@@ -364,6 +392,11 @@ public Authentication autoLogin(HttpServletRequest request, HttpServletResponse
         return new SecurityComponents( findBean(AuthenticationManager.class, context), uds, rms);
     }
 
+    @Override
+    public Filter createFilter(FilterConfig filterConfig) {
+        return new ChainedServletFilter(super.createFilter(filterConfig), new HttpHeaderFilter(this));
```

Comment:
so the filter is created regardless of if you want to use this new functionality or not?

(I don't recall if the filter only gets created when changing the realm, and not when just reconfiguring it)

Reply:
Yes it's created regardless.